docs/pages/more/expo-cli.mdx

```diff
@@ -315,6 +315,26 @@ The command `npx expo install expo-camera` and `npx expo install expo-camera --f
 
 <Terminal cmd={['$ npx expo install --fix']} />
 
+### Configuring dependency validation
+
+> Available in SDK 49 and above.
+There may be circumstances where you want to use a version of a package is different from the version recommended by `npx expo install`.
```

Comment:
@keith-kurak, is this line supposed to be part of the callout? 

<img width="880" alt="CleanShot 2023-06-11 at 23 33 02@2x" src="https://github.com/expo/expo/assets/10234615/ac4e5d7b-8e7e-43fa-af1f-2245726e6e04">

Reply:
Nope, didn't mean that! Added a line break to fix